doc/rados/operations/bluestore-migration.rst

```diff
@@ -2,58 +2,62 @@
  BlueStore Migration
 =====================
 
-Each OSD can run either BlueStore or FileStore, and a single Ceph
+Each OSD can run either BlueStore or Filestore, and a single Ceph
 cluster can contain a mix of both.  Users who have previously deployed
-FileStore are likely to want to transition to BlueStore in order to
-take advantage of the improved performance and robustness.  There are
+Filestore OSDs should transition to BlueStore in order to
+take advantage of the improved performance and robustness.  Moreover,
+Ceph releases beginning with Reef do not supper Filestore. There are
```

Comment:
s/supper/support/

Reply:
That's what happens when I edit while hungry.